docs/Comparison.md

```diff
@@ -65,13 +65,12 @@ It should be easy to convert a Helm chart into a FrameworkVersion since we can j
 
 ## Comparison Table
 
-
-| Project | Definition Language |Uses CRDs |  Dependencies | Multi Step |  Parameters |  Custom Lifecycles | Install Component| App Repo|
-|---------|----|-------|-----------|--|-------------------|--------------|--| --|
-|Raw Yaml |YAML| No | No | No | No | No| No | No|
-|Helm 3 | Lua | No | Yes | No | Yes | Yes | CLI | Yes |
-|Helm 2|  Also lua? |No | ?? | No | Yes | No | CLI + Tiller | Yes|
-| ksonnet|  jsonnet/libsonnet | No | ?? | No | Yes | ?? | CLI| No |
-| OpenShift Templates| yaml| No | No | No | Yes | No | Just Openshift....| No |
-| Kustomize | yaml | No | No | Yes | No | No| CLI| No|
-|KUDO| Yaml + Kustomize | Yes | Yes | Yes |Yes | Yes | Yes | Yes|
```

Comment:
btw... the core issue with table is that it requires 3 `-` for a min `---`  between columns.  we do not need to maintain all the `-`s, I just think it looks nicer when editing.

Reply:
we might also need to play with the padding https://github.com/kudobuilder/kudo/blob/4a79187e90f383d805b405f797b684147225d8d6/site/themes/book/assets/_variables.scss#L3 e.g to `3.5` or so.